docker-compose.yml

```diff
@@ -39,9 +42,64 @@ services:
         extra_hosts:
             # just if you have a local CKAN instance running in _nginx_ host in you local machine
             # export HOST_IP=`ip -4 addr show scope global dev docker0 | grep inet | awk '{print \$2}' | cut -d / -f 1`
-            - "nginx:${HOST_IP}"
+            - "ckan:${HOST_IP}"
         healthcheck:
             test: ["CMD-SHELL", "[ -f /usr/local/airflow/airflow-webserver.pid ]"]
             interval: 30s
             timeout: 30s
             retries: 3
+
+    ckan-dev:
+        build:
```

Comment:
no building, please use official images

Reply:
Still building here, please remove all Dockerfiles from this repo (other than the ckan-ng-harvester one)

Reply:
All removed. No more builds